File: certingon.employess.api/certingon.employess.api/Model/Employee.cs
using System.Xml.Serialization;

namespace certingon.employess.api.Model
{
    [XmlRoot("Employee")]
    public class Employee
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string City { get; set; }
        public Department Department { get; set; }
        public bool IsActive { get; set; }
    }

    public enum Department
    {
        Development = 0,
        Management = 1,
        HR = 2
    }
}

File: certigon.employees.web/src/app/models/employee.ts
export class Employee {
    constructor(init?: Partial<Employee>) {
      Object.assign(this, init);
    }
  
    id?: number;
    name = "";
    surname = "";
    city = "";
    department?: Department;
    isActive?: boolean;
  }
  
  export enum Department {
    Development = 0,
    Management = 1,
    HR = 2,
  }
  